cdn/cdn_test.go

```diff
@@ -4,10 +4,20 @@ import (
 	"context"
 	"fmt"
 	"log"
+	"os"
+	"testing"
 
 	"github.com/Azure-Samples/azure-sdk-for-go-samples/helpers"
 )
 
+func TestMain(m *testing.M) {
+	err := helpers.ParseArgs()
```

Comment:
Given that this ends up looking so much like the default implementation of TestMain, I'd add a comment here mentioning that ParseArgs allows us to target the particular region that we need.

Reply:
Either that or we move away from command line args for this and move to environment variables for everything.

Reply:
CLI is beautiful.

Reply:
Fixed